web/pdf_find_controller.js

```diff
@@ -62,6 +62,421 @@ function normalize(text) {
   return [normalizedText, diffs];
 }
 
+const CHARACTERES_ACCENTED_MAP = {
+  À: "A",
+  Á: "A",
+  Â: "A",
+  Ã: "A",
+  Ä: "A",
+  Å: "A",
+  Ấ: "A",
+  Ắ: "A",
+  Ẳ: "A",
+  Ẵ: "A",
+  Ặ: "A",
+  Æ: "AE",
```

Comment:
As mentioned previously, "expanding" a character in this way will break highlighting of search results.